charts/velocity-registry/values.yaml

```diff
@@ -0,0 +1,43 @@
+replicaCount: 1
+
+image:
+  repository: registry
+  tag: 2.8.1
+  pullPolicy: IfNotPresent
+
+env:
+  RegistryAuth:
+    name: REGISTRY_AUTH
+    value: htpasswd
+  RegistryAuthRealm:
```

Comment:
please be consistent with all value naming. should start with a lowercase letter and then camelcase, like `registryAuth`